### PR TITLE
Add bazel BCR config

### PR DESCRIPTION
**Pull request by keith**
This is based on sourcekitten's

### CODE REVIEW

File: .bcr/source.template.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "url": "https://github.com/realm/SwiftLint/releases/download/{TAG}/bazel.tar.gz",
```
Comment (keith) [inline .bcr/source.template.json:2]: I assume the bazel archive is the right one?

Comment (jpsim) [reply .bcr/source.template.json:2]: Yes

**Review by jpsim — APPROVED** [review]

**Comment by jpsim** [comment]
Oh I think I’ll need to add a fixed releaser so the BCR app uses my fork of the registry.

**Comment by jpsim** [comment]
36c24c822c0944813ad841b18e3028d7ce17493f

**Pull request merged**